src/UI/Implementation/Component/Dropdown/Renderer.php

```diff
@@ -10,6 +10,12 @@
 use ILIAS\UI\Implementation\Render\ResourceRegistry;
 
 class Renderer extends AbstractComponentRenderer {
+
+	/**
+	 * @var string
+	 */
+	private $class_name = 'dropdown';
```

Comment:
We do not want CSS-Class names to appear in PHP. Please stop using this pattern.